src/main/java/net/sf/jabref/gui/EntryTypeList.java

```diff
@@ -127,16 +126,6 @@ public void addDefaultActionListener(ActionListener l) {
     }
 
     @Override
-    public void actionPerformed(ActionEvent e) {
-        // Default button pressed.
-        if (e.getSource() == def) {
-            def.setEnabled(false);
-        } else {
-            super.actionPerformed(e);
```

Comment:
The refactoring is ok since the super call is also refactored in the same commit (the super class does not implement ActionListener anymore and thanks to the inheritance the super call is now useless). 

I looked at the type hierarchy of these two classes to check that this change has no impact on other classes.